configdialog.cpp

```diff
@@ -529,12 +531,17 @@ void PaletteCustomizeDialog::delete_current_palette() {
 	}
 }
 
+std::wstring PaletteCustomizeDialog::get_png_export_path(const std::wstring &base_path, const std::wstring &palette_name) {
+	return std::format(L"{}\\yokins-{}-{}", base_path, palette_name, rand());
```

Comment:
Suffixing the folder with a random number is to ensure that exporting a palette with a duplicate name won't cause any overwriting (or at least that it will be highly unlikely)

Reply:
So you're saying there's a chance...

Reply:
Well, I found another oversight.
This isn't seeded, so it gives the exact same numbers every time.